### ISSUE TITLE
Update the "Import workspace" dialog

### ISSUE THREAD

**Issue by ggeisler** [ProjectMirador/mirador#2290]
Related to #2286

This dialog is not currently in `master` but was added in @glefi's [PR #2002](https://github.com/ProjectMirador/mirador/pull/2002) so the updates below could be added before completing the PR.

Updates needed:
- Should follow the approach used in the [form dialogs example](https://material-ui.com/demos/dialogs/#form-dialogs)
- Heading should be changed to match the menu label: "Import workspace"
- Heading should be changed to use an `H2`
- "Cancel" button should be added as a [text button](https://material-ui.com/demos/buttons/#text-buttons)
- "Import workspace" button should be right-aligned and re-labeled "Import"
- "Import" button should be a [contained button](https://material-ui.com/demos/buttons/#contained-buttons)
- Add hint text below the input area to give the user a better idea of what is expected: `Paste a Mirador 3 configuration to be imported`
- Widen the input area (~400px) so the pasted content doesn't have to wrap as much

Result should look roughly like this (heading font should look a bit different than this when changed to an `h2`):

<img width="463" alt="import" src="https://user-images.githubusercontent.com/101482/54790843-ddfd9e00-4bf4-11e9-8661-bb534207bda6.png">

**Comment by glefi** [comment]
@ggeisler I wonder if it may cause problems regarding the responsiveness of the ui, if the width is defined using pixels. Maybe we should consider using percentages. What do you think?

**Comment by ggeisler** [comment]
@glefi Yeah, I understand your concern. But we might have the same problem using percentages, too. For example, a percentage of say, 50% might work on a desktop at 1000px or 1200px wide (dialog is then 500px or 600px wide), but then on a phone that is say, 380px wide, the dialog will be only 190px wide, which is pretty narrow.

It seems like the best solution might be to use different widths at different responsive breakpoints. If this chart reflects the breakpoints used in the app (https://material.io/design/layout/responsive-layout-grid.html#breakpoints) then maybe we go with something like this:

xsmall - 90%
small - 50%
medium - 33%
large - 25%

Or if we use fixed pixel sizes:

xsmall - 300px
small and larger: 400px

**Comment by cbeer** [comment]
This is in done, but isn't closed? I'm not seeing the described behavior in master, so maybe moving it to done was a mistake?

**Comment by glefi** [comment]
@cbeer Thanks for your note! I had mistakenly assumed, that the changes had been already merged to the 'initial config import' branch and therefore to master.
